authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -750,6 +751,15 @@ public OAuth2AuthorizedClientManager authorizedClientManager(
       return manager;
     }
 
+    @Bean
+    public OidcUserService oidcUserService() {
+      final var userService = new OidcUserService();
+      // disable fetching user info from the IdP, as we never use it, and this can cause rate
+      // limiting issues with some IdPs
+      userService.setRetrieveUserInfo(ignored -> false);
+      return userService;
+    }
```

Comment:
🔧 Can we have a test to validate this behaviour? There could be some examples of testing structure in the `OidcFlowTest` class